README.md

```diff
@@ -1,164 +1,76 @@
-# OpenPubkey Reference Implementation
+# OpenPubkey
 
-This repo contains the current reference implementation of OpenPubkey. The reference implementation is a work in progress.
+## Overview
 
-## What is OpenPubkey?
+OpenPubkey allows any OpenID Connect (OIDC)-speaking identity provider to attest to cryptographic public keys alongside identities. This establishes cryptographic signing keys at scale for users and workloads, in a manner that is scalable for workloads and seamless and invisible to users. OpenPubkey does not require any changes to OpenID Connect so it works with today’s OpenID providers.
 
-OpenPubkey adds user or workload generated public keys to OpenID Connect (OIDC) enabling identities to sign messages or artifacts under their OIDC identity. In essence, OpenPubkey is a protocol for getting OpenID Providers (OPs) to bind identities to public keys.
+OpenPubkey works by essentially turning an OpenID Provider (OP) into a Certificate Authority (CA). (A CA is a trusted entity that issues certificates that cryptographically bind an identity with a cryptographic public key.) To do this, OpenPubKey introduces a new cryptographic object — the PK Token — that binds an identity (i.e., a user or a workload) to its public key.
 
-Verifiers can check that these signatures are valid and associated with the signing OpenID identity. OpenPubkey does not add any new trusted parties beyond what is required for OpenID Connect. It is fully compatible with existing OpenID Providers (Google, Azure/Microsoft, Okta, OneLogin, Keycloak) without any changes to the OpenID Provider.
+The PK Token and the public key can then be used to enable a myriad of applications. For example:
 
-### ID Tokens and PK Tokens
+✍️ [Docker, Inc](https://www.linkedin.com/company/docker/) will use OpenPubKey to sign containers in a public container registry.
```

Comment:
What is the intent of this character  ✍️?

Why link to company LinkedIn pages rather than the company home pages?

Reply:
It was meant to reference the "sign" action. I've [reworded a bit](https://github.com/openpubkey/openpubkey/pull/83/commits/85ca102ebe604104a3bfdaeeb88dcf6e05c794df) and went with standard bullets for easier reading.